src/encoded/tests/test_search_views.py

```diff
@@ -436,6 +436,32 @@ def test_search_views_matrix_response_no_results(workbook, testapp):
     assert r.json['notification'] == 'No results found'
 
 
+def test_chip_seq_matrix_view(workbook, testapp):
+    res = testapp.get('/chip-seq-matrix/?type=Experiment').json
+    assert res['@type'] == ['ChIPSeqMatrix']
+    assert res['@id'] == '/chip-seq-matrix/?type=Experiment'
+    assert res['@context'] == '/terms/'
+    assert res['notification'] == 'Success'
+    assert res['title'] == 'ChIP Seq Matrix'
```

Comment:
ChIP-seq matrix

Reply:
Called this "ChIP-seq" to match the title in the backend. Let me know if you think the backend should be re-named this.